client/app/components/routes/access-requests/AccessRequests.tsx

```diff
@@ -0,0 +1,78 @@
+import { GridRowsProp } from "@mui/x-data-grid";
+
+import { actionHandler } from "@/app/utils/actions";
+import DataGrid from "@/app/components/datagrid/DataGrid";
+import { UserOperator } from "./types";
+
+// 🛠️ Function to fetch user-operators
+async function getUserOperators() {
+  try {
+    return await actionHandler(
+      "registration/user-operators",
+      "GET",
+      "/dashboard/operators",
+    );
+  } catch (error) {
+    // Handle the error here or rethrow it to handle it at a higher level
+    throw error;
+  }
+}
+
+function capitalizeString(value: string): string {
+  return value.slice(0, 1).toUpperCase() + value.slice(1);
+}
+
+// 🧩 Main component
+export default async function AccessRequests() {
+  // Fetch userOperator data
+  const userOperators: [UserOperator] = await getUserOperators();
+  if (!userOperators) {
+    return (
+      <div>
+        No user-operators data in database (did you forget to run `make
+        loadfixtures`?)
+      </div>
+    );
+  }
+
+  // Transform the fetched data into rows for the DataGrid component
+  const rows: GridRowsProp =
+    userOperators.length > 0
+      ? userOperators.map(
+          ({ id, status, first_name, last_name, email, legal_name }) => {
+            return {
+              id,
+              status: capitalizeString(status),
```

Comment:
do we still need capitalize string? I saw the statuses are now capitalized in the mock data, which makes me think they're capped in the db too

Reply:
Removed

Reply:
J I need to add this back in it is still needed.